contracts/Datastore.sol

```diff
@@ -15,9 +15,9 @@ contract Datastore is AragonApp {
     using GroupLibrary for GroupLibrary.GroupData;
 
     bytes32 constant public DATASTORE_MANAGER_ROLE = keccak256("DATASTORE_MANAGER_ROLE");
```

Comment:
I can not comment above, but I think you can also remove `ACL` and `ACLSyntaxSugar` from this contract too.

Reply:
Oh right good catch! I removed those imports